codersdk/deployment.go

```diff
@@ -1474,6 +1475,16 @@ when required by your organization's security policy.`,
 			Group:       &deploymentGroupOIDC,
 			YAML:        "signupsDisabledText",
 		},
+		{
+			Name: "Skip OIDC issuer checks (not recommended)",
+			Description: "OIDC issuer urls must match in the request, the id_token 'iss' claim, and in the well-known configuration. " +
+				"This flag disables that requirement, and can lead to an insecure OIDC configuration. It is not recommended to use this flag.",
+			Flag:  "oidc-skip-issuer-checks",
+			Env:   "CODER_OIDC_SKIP_ISSUER_CHECKS",
```

Comment:
Can we add the `DANGEROUS_` prefix? Because it is.

Reply:
Added!